test cases/frameworks/4 qt/manualinclude.cpp

```diff
@@ -10,9 +10,14 @@ class MocClass : public QObject {
     Q_OBJECT
 };
 
+void testSlot() {
+	;
+}
+
 int main(int argc, char **argv) {
     ManualInclude mi;
     MocClass mc;
+    QObject::connect(&mi, &ManualInclude::mysignal, &testSlot);
```

Comment:
This doens't work with Qt4, you may use old Qt4 connect style. You can move the test slot in MocClass. Then something like this will work:
```
QObject::connect(&mi, SIGNAL(ManualInclude::mysignal()), &mc, SLOT(testSlot()));
```